config/envs/production.yaml

```diff
@@ -168,6 +172,12 @@ servers:
           resources:
             requests:
               storage: "1Gi"
+
+      - metadata:
+          name: "{{.ApplicationName}}-h2-claim-{{$index}}"
```

Comment:
Please review this PVC in all env files. Is one needed in 3+ environments? If not, let's remove it from common.yaml. If yes, it should probably have a better name.

Reply:
its only needed for authoring... ill make the changes.

Reply:
done